nsv13/code/modules/overmap/fighters/fighters.dm

```diff
@@ -552,64 +263,96 @@ You need to fire emag the fighter's IFF board. This makes it list as "ENEMY" on
 		prebuilt_setup()
 	dradis = new /obj/machinery/computer/ship/dradis/internal(src) //Fighters need a way to find their way home.
 	update_stats()
-	fuel_setup()
 	obj_integrity = max_integrity
 	RegisterSignal(src, COMSIG_MOVABLE_MOVED, .proc/check_overmap_elegibility) //Used to smoothly transition from ship to overmap
 	RegisterSignal(src, COMSIG_AREA_ENTERED, .proc/update_overmap) //Used to smoothly transition from ship to overmap
 	add_overlay(image(icon = icon, icon_state = "canopy_open", dir = SOUTH))

```

Comment:
Should the dir align with the fighter?

Reply:
Maybe? We aren't currently changing the dir, but just rotating them or something like that?